File: Cats and a Mouse/cats-and-mouse.py
def catAndMouse(x, y, z):
    """
    :rtype x: int
    :rtype y: int
    :rtype z: int
    :return: str
    """
    return ['Cat A', 'Cat B', 'Mouse C'][0 if abs(x - z) < abs(y - z) else 1 if abs(y - z) < abs(x - z) else 2]


print(catAndMouse(1, 2, 3))  # 'Cat B'
print(catAndMouse(1, 3, 2))  # 'Mouse C'

File: Cats and a Mouse/cats-and-mouse.js
/**
 * Cat and mouse function
 *
 * @return {string}
 * @param {number} x
 * @param {number} y
 * @param {number} z
 */
function catAndMouse(x, y, z) {
  if (Math.abs(x - z) < Math.abs(y - z)) {
    return 'Cat A';
  } else if (Math.abs(y - z) < Math.abs(x - z)) {
    return 'Cat B';
  } else {
    return 'Mouse C';
  }
}

console.log(catAndMouse(1, 2, 3)); // 'Cat B'
console.log(catAndMouse(1, 3, 2)); // 'Mouse C'